assemble/pom.xml

```diff
@@ -106,11 +106,6 @@
       <artifactId>jaxb-core</artifactId>
       <optional>true</optional>
     </dependency>
-    <dependency>
-      <groupId>commons-beanutils</groupId>
-      <artifactId>commons-beanutils</artifactId>
-      <optional>true</optional>
-    </dependency>
```

Comment:
Can probably also remove the entry from `assemble/src/main/assemblies/component.xml`